docs_manual/source/ext_tool_cyberduck.rst

```diff
@@ -0,0 +1,56 @@
+.. _ext_tool_cyberduck:
+
+Cyberduck
+^^^^^^^^^
+
+Cyberduck is a libre network storage browser for Mac and Windows.
+It can be used to access the iRODS filesystem on SODAR directly.
+
+Installation & Setup
+====================
+
+1. Download the Cyberduck installer from this website:
+https://cyberduck.io/download/
+
+.. note::
+    While Cyberduck is generally free to use, it will ask you to
+    donate to the project by making a one-time license key purchase.
+
+2. Download the SODAR iRODS connection profile:
+   :download:`iRODS (CUBI SODAR).cyberduckprofile <_static/ext_tool_cyberduck/iRODS (CUBI SODAR).cyberduckprofile>`
+
+3. Install and start Cyberduck. Your screen should look like this:
+
+   .. image:: _static/ext_tool_cyberduck/Cyberduck_home_empty.png
+
+4. Import the downloaded Cyberduck profile by double-clicking or dragging it
+   onto the Cyberduck window. A new dialog should appear. Fill in your
+   SODAR username and password in the respective fields.
+
+   .. image:: _static/ext_tool_cyberduck/Cyberduck_new_profile.png
+
+
+Using Cyberduck
+====================
+
+1. Connect to SODAR iRODS by double-clicking on the newly saved bookmark.
+
+   .. image:: _static/ext_tool_cyberduck/Cyberduck_home_profile.png
+
+2. The file browser should list collections for all projects you have access too.
```

Comment:
Please ensure max. line length of 80 (long rst tags such as the earlier download link can of course be excluded).